docs/examples/extended-secret/certificate.yaml

```diff
@@ -0,0 +1,112 @@
+    XTonpFG9TQkFqhWPvEryYKs9QUkR3mavZ4feHL22Jmuk3R96i1FngFe9Dx21pHZT
+    LnhVYQVHB7UuM8Xrm5yI9cO5TFxq6JvOpK4LiSNFHSvT1qajoV1uFiVSSXUffICY
+    Qw9KJDlWCwKsBFwZP5rURoP/dGiIviWqGSr/lXmusja84eCLLOtsJyiYwnZSY2Ph
+    U+LzRyEDiIZim5BuMZ35oBKgDENDmOCbBfGoMC0RyYZP/LVX4s6Aq3iOpm73iUZg
+    rJni+HikPRxw3zm41r15Ja9QQCGC975dhWXpiBE5a+RQ/hou9kZiKXy1qdYexd3K
+    OnsiyPABImaI2ec+mRAEpxsIDm2Hs9Lrf8BfgNLsGkSsqan8ztk+Ok8Y1M9KPYw9
+    2h8+VD3/GVcM/LUPSxq5u/fLuKkGFwQ0Nr9GfFahXOKHhEkR1IK/pB2aCnOm0CIP
+    O8wngwr6hEhJqqgsyNj9aBtjXKQBEmBbuOvz5bH63w5/kJFy5/Bpc+JXhqOlDk1E
+    CDGEIgI0JWlpqNeXqscFCR8rRKGZXuGs5d+1F5Zy771nbOU7yHUCggEBAPArLpds
+    pb3QaZeD4fEcXk1RzB+Ni8EhWDMHGR7PwDixkOGGY7q+SgUkPhfsmE380F4AVb/H
+    fufd5zgo11G1P4K45lswNAzM4Upp6KSSH6TYKy0BddDn/fJcopz3BqQ988tCmEPt
+    4sNCQM8dHTENkTXAMuEKzS2Q1DvCSyDda9+AotF77NMPsd9FyvzdTOILM6DcbYhJ
+    GNVjHk22J0odFOeVjUtLeqFOwSKKP5CS+Csqw2OL7MxzHU8K/qQcCsI7JXVFkL5D
+    MG/lVnUgf/vqhJQJUYJgjVNp9Xl/6xMPFDNZlwpAO8w+l66gr0r1pIzbOsNl6eZk
+    SjSAsa3k88SRJHcCggEBANifToK4l4ee3XrICvOZrXrEhTdXg/yiC4E+VszJwdf7
+    2MoqOWMT1m8y5IUUorEPL+qSgR7UsJ89VZAPAo74REk3+801oF50yKoxLDK2xZiN
+    rB7CNujYpR3yVg4VRVPc20RY3xZJDGKOWFeGDs9ba1c9cq9THQkNf1/b8Q61rB7U
+    iQe3nhmmrNGXIp6Jv20PNEhIDtViurmzT7EdqDTuLI34+FmfaC9nZVruW/SqtN4e
+    7aPAj0pRUwzfUkwZc77W5W5i0BY2nOz3gfD9Zwx1OBmGvLcfifMNmerfAE0F8qWd
+    1j4u9vzAD5Gm86PJm7UPKkwZh7RwFbjOd7iinTB4xgMCggEAC67OEWejmFwvBsPP
+    ld7/Z1t3RoGoLrjkhXbu7eTFQPzCQ3Y6G7dt+loM+zTzI7gMmsYBZdbagYT13nF+
+    J+ddM4qzCfSKU6UTwH7uZjMFfwtU7leH16WxYLzI3EHX/N1WoqiWp5dG1KDAblLI
+    PL6gNbEnyHKyEGlfKZkF9o9O1Ax53x7cIlg9GtNWUH6jYsuQs16FpX+IybiMnNMj
+    jiNkBO/tc6A0Uux10KxQ80Ny9waZgpon5roiztmGEY2bqF8gL1g29kpmbMsHLaNr
+    FmIOGPVrFaCEv24EFuj2EnNFVEojYshFIGZbEo+Wd1sDqiZv8Vlfy2lNgshP3Fmq
+    0gENeQKCAQBIuWLwO+yTWWBdwh77HVLCbouSwszK2dQm9bjKU8nCKwjjBj3x500M
+    U/eoPf9IliKuC7VRlnIdY8f/7yoauXqjYiolAKXwUCC2EoAWHopD5vwr72Pj1CfK
+    3/rIC5gUyYbLcVKj9q4zwDokWBtMvhJRXY3NEogi6+chVYOePpukVhITBzAKyIwo
+    aSb0NqaPhJU21woz1Ut+4JYWjjBUZMmMeFNYM04U7A948aDpiQ3jrFp3qUPx3U+W
+    eLr2VW0oWpj38DCP5JHWdFmpgElCQ3AsEbH1gJ8dfCl6UpeVRDPsKgstITNGM2yG
+    p5gQGrUtmgPHPKL0jtScEKTc1asUBNflAoIBAQDiNv35TrXj2T7iUWZ193k+uIt4
+    kmfDodHmdk5gL8i7203XGc3UWusVoBwUn6RYO02tUmdYUYrf5qXM6a1IKbLxZvp1
+    Z9vBFldumr97yoEm6cacqWe3Wz8nHO0e0zqaOkdpm64cdjQ7sy1stedjEjBFEs+c
+    8wuIeKREgYbamLrGO694clxV2E5jZxFiyDmZaWPoVWFGvdEq2okRnLmVcAUcKAnn
+    6pxd+7ZT1Ibxt70Fum1G9qRJzlzbxVQHx7LzEUOD3T1TWhGFLNDVhS0A75ohF7OA
+    wvHxHWtIIkR7z2AEPlbmx2VboheAh5eqjOjtxeUettJAMiINCMkBIWVN+FaU
+    -----END RSA PRIVATE KEY-----
+---
+apiVersion: fissile.cloudfoundry.org/v1alpha1
+kind: ExtendedSecret
+metadata:
+  name: generate-certificate
+spec:
+  request:
+    certificate:
+      CAKeyRef:
+        Key: private_key
+        Name: cert-ca
+      CARef:
+        Key: certificate
+        Name: cert-ca
+      alternativeNames:
+        - foo.com
+        - '*.foo.com'
+      commonName: routerSSL
+      isCA: false
+      signerType: external
```

Comment:
This looks like it's not using the API server to sign

Reply:
Removed CA infos. Sorry for misleading example.